CJSpohn/whats-cooking-refactor
# Feature/fetch

Is this a fix or a feature?
Feature

What is the change/fix?
Removed local data and fetched all necessary data with fetch and promise.all(). Broke start up function into separate pieces to be more DRY.

Where should the reviewer start?
scripts.js line 14

How should this be tested?
run npm start in the repo and make sure all prior functionality is retained with the new fetch calls.
